### PR TITLE
Issue #15449: removed whole folder reference and added individual file names to be grepped in google-java-format.sh

### PR DESCRIPTION
**Pull request by Zopsss**
#15449

removed whole folder names from grep list and used individual file names instead.

### CODE REVIEW

File: .ci/google-java-format.sh [hunk 1]
```diff
@@ -4,16 +4,17 @@ set -e
 
 JAR_PATH="$1"
 
-INPUT_PATHS=($(find src/it/resources/com/google/checkstyle/test/ -name "Input*.java" \
+INPUT_PATHS=($(find src/it/resources/com/google/checkstyle/test/ \
+    \( -name "Input*.java" -o -name "InputFormatted*.java" \) \
```
Comment (romani) [inline .ci/google-java-format.sh:8]: this change is not required.
` "Input*.java"` is enough and fully covering `"InputFormatted*.java"`

File: .ci/google-java-format.sh [hunk 2]
```diff
@@ -4,16 +4,17 @@ set -e
...
     | sed "s|src/it/resources/com/google/checkstyle/test/||" \
-    | grep -v "rule711generalform" \
+    | grep -v "rule711generalform/InputSingleLineJavadocAndInvalidJavadocPosition.java" \
```
Comment (romani) [inline .ci/google-java-format.sh:10]: add formatted file for this file, or you plan to do this in separate PR ?

**Review by romani — CHANGES_REQUESTED** [review]
items:

**Comment by romani** [comment]
can you write bash block that find all Input file that are referenced in this script and make sure we have InputFormatted file for it ?
or it is too advanced bash skills ?

File: .ci/google-java-format.sh (hunk 1 above)
Comment (romani) [reply .ci/google-java-format.sh:8]: Please do this

File: .ci/google-java-format.sh (hunk 2 above)
Comment (romani) [reply .ci/google-java-format.sh:10]: Ok, it will be separate activity

File: .ci/google-java-format.sh (hunk 1 above)
Comment (Zopsss) [reply .ci/google-java-format.sh]: done
